app/scripts/metamask-controller.js

```diff
@@ -1799,9 +1798,8 @@ export default class MetamaskController extends EventEmitter {
           `${this.loggingController.name}:add`,
         ],
       }),
-      isEthSignEnabled: () =>
-        this.preferencesController.store.getState()
-          ?.disabledRpcMethodPreferences?.eth_sign,
+      // TODO remove this option
+      isEthSignEnabled: () => false,
```

Comment:
Can we remove this now?

Reply:
Yes. Not sure why but I thought I'd forgotten to remove it as an option so was going to loop back to it but it can be remove removed now.

Reply:
https://github.com/MetaMask/metamask-extension/pull/24756/commits/343ee961208c68f42397d6591f73f14d2c74891e